packages/medusa/src/api-v2/admin/promotions/rule-operator-options/route.ts

```diff
@@ -0,0 +1,32 @@
+import { PromotionRuleOperator } from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../types/routing"
+
+export const GET = async (
+  req: AuthenticatedMedusaRequest,
+  res: MedusaResponse
+) => {
+  const operators = [
```

Comment:
discussion: any reason why we are not using the RuleOperato instead? looks like the promotion rule operator is the same and could be removed to ensure that we are using all the same operators everywhere

Reply:
you're right. Changing it!